tests/test_api.py

```diff
@@ -6,13 +6,19 @@
 
 from api import app, configure_api_app
 from gazettes import GazetteAccessInterface, GazetteRequest
+from suggestions import SuggestionService
 
 
 @GazetteAccessInterface.register
 class MockGazetteAccessInterface:
     pass
 
 
+@SuggestionService.register
+class MockSuggestionService:
+    pass
+
+
 class ApiGazettesEndpointTests(TestCase):
```

Comment:
I think we should have an `ApiSuggestionEndpointTests` as well.

Some suggestions of what we could include (feel free to ignore, change or add stuff):

- `test_api_should_fail_when_try_to_set_any_object_as_suggestion_service`
- `test_api_should_not_fail_when_set_correct_suggestion_service`
- `test_suggestion_endpoint_should_fail_when_request_body_is_not_json`
- `test_suggestion_endpoint_should_fail_when_title_is_missing`
- `test_suggestion_endpoint_should_fail_when_email_address_is_missing`
- `test_suggestion_endpoint_should_fail_when_email_address_is_invalid`
- `test_suggestion_endpoint_should_fail_when_content_is_missing`